src/config.rs

```diff
@@ -356,6 +356,10 @@ pub struct DbOptions {
     /// you may benefit from setting this higher (if you have enough memory available)
     pub filter_bits_per_key: u32,
 
+    /// The max number of unflushed WAL SSTables. Writes will be paused if there are
+    /// more unflushed WAL SSTables than this value.
+    pub max_unflushed_wal: usize,
```

Comment:
can you add this new field to the examples of configuration files in this file's header?

Reply:
Done! 3db24e8